packages/radio/src/css/index.css

```diff
@@ -0,0 +1,92 @@
+.psds-radio-group:focus {
+  outline: 0;
+}
+.psds-radio-group--disabled {
+  opacity: 0.5;
+}
+.psds-radio-group__button-container {
+  padding: var(--ps-layout-spacing-xxsmall) 0;
+}
+.psds-radio-group__label {
+  color: var(--ps-colors-text-icon-high-on-dark);
+  font-size: var(--ps-type-font-size-100);
+  font-weight: var(--ps-type-font-weight-500);
+  margin-bottom: var(--ps-layout-spacing-xxsmall);
+}
+.psds-radio-group__label.psds-theme--light {
+  color: var(--ps-colors-text-icon-high-on-light);
+}
+.psds-radio-group__sub-label {
+  color: var(--ps-colors-text-icon-low-on-dark);
+  font-size: var(--ps-type-font-size-100);
+  font-weight: var(--ps-type-font-weight-default);
+  margin-top: var(--ps-layout-spacing-xxsmall);
+}
+.psds-radio-group__sub-label.psds-theme--light {
+  color: var(--ps-colors-text-icon-low-on-light);
+}
+.psds-radio-button {
+  display: flex;
+  align-items: center;
+  padding: var(--ps-layout-spacing-xxsmall) 0;
+  cursor: pointer;
+}
+.psds-radio-button--disabled {
+  cursor: not-allowed;
+}
+.psds-radio-button__halo {
+  transform: translateY(3px);
+}
+.psds-radio-button__circle-outer {
+  width: 16px;
+  margin-right: var(--ps-layout-spacing-small);
+  pointer-events: none;
+}
+.psds-radio-button__circle {
+  position: relative;
+  display: inline-block;
+  height: 16px;
+  width: 16px;
+  border-radius: 50%;
+  border: 2px solid var(--ps-colors-text-icon-low-on-dark);
+  background: var(--ps-colors-background-dark-1);
+}
+.psds-radio-button__circle:focus {
+  outline: 0;
+}
+.psds-radio-button__circle.psds-theme--light {
+  background: var(--ps-colors-background-light-3);
+  border-color: var(--ps-colors-text-icon-low-on-light);
+}
+.psds-radio-button__circle--checked, .psds-radio-button__circle--checked.psds-theme--light {
+  border-color: var(--ps-colors-blue-6);
+  background: var(--ps-colors-blue-6);
+}
+.psds-radio-button__circle-inner {
+  display: block;
+  height: calc(100% - 6px);
+  width: calc(100% - 6px);
+  margin: 3px;
+  border-radius: 50%;
+  background: var(--ps-colors-white);
+}
+.psds-radio-button__input {
```

Comment:
opportunity to re-use the core/accessibility/.screenreader-only selector, if you'd like.